client/src/components/App.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import axios from 'axios';
+import Namelist from './Namelist.jsx';
+
+class App extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            userPictures: [],
+            likes: 0,
+            likeWord: 'likes',
+
+
+        }
+        this.onClickLikes = this.onClickLikes.bind(this);
+        this.onClickProfiles = this.onClickProfiles.bind(this);
+
+    }
+
+    componentDidMount () {
+        axios.get('/userinfo')
+            .then( (data) => {
+                var array = [];
+                for(var i = 0; i<=8; i++){
+                    array.push(data.data[i])
+                }
```

Comment:
Cool idea to store only the first 8.  I wonder if that optimizes some things.  Is there any circumstance on the actual site where you would need to view more than 8?